src/sentry/static/sentry/app/views/admin/adminBuffer.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import InternalStatChart from 'app/components/internalStatChart';
+
+const AdminBuffer = () => {
+  const since = new Date().getTime() / 1000 - 3600 * 24 * 7;
+
+  return (
+    <div>
```

Comment:
should we replace this `div` with `React.Fragment`?

Reply:
Maybe, not sure, I didn't look too closely :)